### DISCUSSION TITLE
Chia farming without private key

### DISCUSSION THREAD

**Discussion by lebedevilya** [Chia-Network/chia-blockchain#9383] [Q&A]
Hello everyone! First time here and i need your help. I have a remote PC(ubuntu) and im trying to setup it for Chia farming. First I created my Chia account and wallet at local PC and got a private key. Successfully created plotnft and got everything to plot and farm locally. I tested it and it works like a charm, already got some payouts from pool. So now im trying to do the same at remote PC but i dont want to share my private key to remote PC because of security measures(other people can access this remote PC). Using farmer public key and pool public key i can create plots at remote machine and i had already created some plots successfully, but i cant figure out how to farm them.  So, the question is, is it possible to farm without private key? As far as i understand it is possible to farm OG plots without my private key if i configure xch_target_address option at config.yml directly to my wallet. But in order to farm NFT plots i need to add plotnft first and i cant figure out how to do that without entering my private key at this remote machine. If someone has any knowledge about these things and you could help me i would be very grateful. Thanks!

**Comment by rmica** [comment]
Harvester: no private key
Node: no private key
Wallet: private key
Farmer: private key
Wallet and Farmer boxes must be secured.

In terms on CLI, the box[es] where you run:
chia start [all] [wallet[-only]] [farmer[-only]]
must be accessible only to trusted users.
